dm/master/coordinator/coordinator.go

```diff
@@ -421,7 +429,7 @@ func (c *Coordinator) restartMysqlTask(w *Worker, cfg *config.MysqlConfig) bool
 		Op:     pb.WorkerOp_StartWorker,
 		Config: task,
 	}
-	resp, err := w.OperateMysqlWorker(context.Background(), req, time.Second*5)
+	resp, err := w.OperateMysqlWorker(context.Background(), req, restartEtcdTimeout)
```

Comment:
Is restarting `Etcd` or `MySQLWorker`?

Reply:
refined to restartMysqlWorkerTimeout in https://github.com/pingcap/dm/pull/453/commits/e6f40e550f45ef72186f6f9853e3e9b31b797b6e